src/code.less

```diff
@@ -25,4 +25,15 @@ pre {
     border-radius: 3px;
     overflow-x: auto;
     white-space: pre-wrap;
```

Comment:
This is a big NO. No wrapping inside `pre` tag.

Reply:
Check the code now

Reply:
I've changed `white-space: pre-wrap` to `white-space: pre` inside `pre code`. Should this also be done for only `pre` tags?

Reply:
Yup.